LibGUI/GTextEditor.cpp

```diff
@@ -163,6 +164,15 @@ void GTextEditor::mousedown_event(GMouseEvent& event)
         return;
     }
 
+    if (m_triple_click_timer.is_valid() && m_triple_click_timer.elapsed() < 250) {
+        m_triple_click_timer = CElapsedTimer();
+        GTextPosition start(0, 0);
+        GTextPosition end(line_count() - 1, m_lines[line_count() - 1]->length());
+        m_selection.set(start, end);
```

Comment:
Isn't triple-click usually "select whole line"?
https://en.wikipedia.org/wiki/Triple-click#Text_fields

Maybe you were using a single-line editor and felt that something wasn't quite right.. :)

Reply:
Correct indeed, I've updated it :)